### DISCUSSION TITLE
Very inconsistent command line parsing

### DISCUSSION THREAD

**Discussion by halloleo** [genotrance/px#156] [Ideas]
I just noticed, that in `px`s cli arguments

> `--config` needs the `=` sign before its value

while 

> `--gateway` works _only_ _without_ the `=` sign, but with a space for its value (plus this argument doesn't seem to be allowed towards the end of the parameter list).

I fully understand that this is not very important (and that it is in principle documented in the help message), but tool beginners like me it causes quite a bit of grief.

So, with a PR I am happy to make it a bit clearer in ReadMe and help page that the user has to be careful with these arguments. 

Let me know.

**Comment by genotrance** [comment]
Some CLI flags are used to set a value like `--proxy` or `--port` but things like `--gateway`, `--hostonly`, `--debug` and others simply turn on a capability. While you need to set them to 0 or 1 in the px.ini file, on the CLI, their presence means 1 and absence means 0 which is why you don't need to specify any value.

**Reply by halloleo** [reply]
Fair enough. Do you min me trying to make this clearer in the usage message and ReadMe?

**Reply by genotrance** [reply]
Yes sure!
